api/endpoints/job.py

```diff
@@ -455,29 +455,74 @@ def get(self, job_id):
 @ns.route('/job/<string:username>/list')
 class Jobs(Resource):
     parser = api.parser()
-    parser.add_argument('page_size', required=False, type=str,
-                        help="Job Listing Pagination Size")
-    parser.add_argument('offset', required=False, type=str,
-                        help="Job Listing Pagination Offset")
+    parser.add_argument('page_size', required=False, type=str, help="Job Listing Pagination Size")
+    parser.add_argument('offset', required=False, type=str, help="Job Listing Pagination Offset")
+    parser.add_argument('job_type', type=str, help="Job type + version (ie. topsapp:v1.0)", required=False)
+    parser.add_argument('tag', type=str, help="User-defined job tag", required=False)
+    parser.add_argument('queue', type=str, help="Submitted job queue", required=False)
+    parser.add_argument('priority', type=int, help="Job priority, 0-9", required=False)
+    parser.add_argument('start_time', type=str, help="Start time of @timestamp field", required=False)
+    parser.add_argument('end_time', type=str, help="Start time of @timestamp field", required=False)
+    parser.add_argument('get_job_details', type=bool, help="Return full details if True. List of job id's if false. Default True.", required=False)
+    parser.add_argument('status', type=str, help="Job status, ie. job-queued, job-started, job-completed, "
```

Comment:
Job statuses accepted here should be OGC Status types and not HySDS status types. 
https://github.com/MAAP-Project/maap-api-nasa/blob/main/api/utils/ogc_translate.py#L22-L54

Reply:
Updated to accept wps/ogc job statuses. Please confirm mapping is correct.